setup.py

```diff
@@ -255,7 +255,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'google-cloud-bigtable>=1.0.0,<2.0.0',
     'google-cloud-container>=0.1.1,<2.0.0',
     'google-cloud-datacatalog>=1.0.0,<2.0.0',
-    'google-cloud-dataproc>=1.0.1,<2.0.0',
+    'google-cloud-dataproc>=2.2.0,<3.0.0',
```

Comment:
Can you add info about this change to [`ADDITIONAL_INFO.md`](https://github.com/apache/airflow/blob/master/airflow/providers/google/ADDITIONAL_INFO.md)?

Reply:
Done in 6162cfe